tools/devops/automation/templates/build/stage.yml

```diff
@@ -19,7 +19,7 @@ jobs:
 - job: configure
   displayName: 'Configure build'
   pool:
-    vmImage: ubuntu-latest
+    vmImage: windows-latest
```

Comment:
Question: If I just merged the PR containing this change, what do I need to do so this PR does not take the credit for this change?

Reply:
well, you do a rebase :)

Reply:
Just merge `main` into this branch:

```
git checkout TurningOnOneLocBuild
git fetch origin
git merge origin/main
```

Reply:
Thank you :) @mandel-macaque @rolfbjarne